Source/Cake.Bumpy/Cake.Bumpy.csproj

```diff
@@ -1,66 +1,39 @@
+    <Copyright>Copyright (c) Cake Contributions 2017 - Present</Copyright>
+    <DocumentationFile>bin\$(Configuration)\Cake.Bumpy.xml</DocumentationFile>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
+    <CodeAnalysisRuleSet>..\Cake.Bumpy.ruleset</CodeAnalysisRuleSet>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>0.0.0</Version>
+    <Authors>fwinkelbauer</Authors>
+    <Description>Cake AddIn for the version management command line tool Bumpy</Description>
+    <PackageLicenseUrl>https://github.com/cake-contrib/Cake.Bumpy/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/cake-contrib/Cake.Bumpy</PackageProjectUrl>
+    <PackageIconUrl>https://cdn.rawgit.com/cake-contrib/graphics/a5cf0f881c390650144b2243ae551d5b9f836196/png/cake-contrib-medium.png</PackageIconUrl>
+    <PackageTags>Cake, Script, Versioning, Build, AssemblyInfo, NuSpec</PackageTags>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
-    <CodeAnalysisRuleSet>..\Cake.Bumpy.ruleset</CodeAnalysisRuleSet>
-    <DocumentationFile>bin\Debug\Cake.Bumpy.xml</DocumentationFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
-    <CodeAnalysisRuleSet>..\Cake.Bumpy.ruleset</CodeAnalysisRuleSet>
-    <DocumentationFile>bin\Release\Cake.Bumpy.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Cake.Core, Version=0.26.1.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Cake.Core.0.26.1\lib\net46\Cake.Core.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="BumpyAliases.cs" />
-    <Compile Include="BumpyRunner.cs" />
-    <Compile Include="BumpySettings.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Cake.Core" Version="0.30.0">
```

Comment:
As a minor comment here...  Since there are no breaking changes between 0.28.0 and 0.30.0 of Cake, I would actually recommend that this be changed to 0.28.0, so that a larger pool of people will be able to consume it.  Unless of course you are taking a dependency on something that specifically ships in 0.30.0.

Reply:
OK, I figured newer was better but this makes sense. Updated the PR